tabular/src/autogluon/tabular/predictor/predictor.py

```diff
@@ -3068,6 +3085,7 @@ def refit_full(self, model: str | List[str] = "all", set_best_to_refit_full: boo
         logger.log(20, f'Refit complete, total runtime = {round(te - ts, 2)}s ... Best model: "{self._trainer.model_best}"')
         return refit_full_dict
 
+    @property
```

Comment:
Maybe we can keep `model_best` as a method as well? Changing it to a property could be potentially confusing since `predictor.model_names` and `predictor.features` are methods, but `predictor.model_best` is now a property.